docs/source/api/core/config.md

```diff
@@ -14,15 +14,10 @@ kernelspec:
   name: vr_python3
 ---
 
-# API reference for `soil` modules
-
-This page contains the detailed documentation of the functions and classes in the
-`virtual_rainforest.soil` modules.
-
-## The `virtual_rainforest.soil.model` module
+<!-- markdownlint-disable MD041 -->
 
 ```{eval-rst}
-.. automodule:: virtual_rainforest.soil.model
+.. automodule:: virtual_rainforest.core.config
```

Comment:
When I build the docs with `sphinx` the API side bar doesn't seem to group functions into categories, just seems to list every function

Reply:
Yeah, need to work out how to nest under modules.

Reply:
All my docs comments here feel like good candidates for a separate pull request, its not like they cause the information contained in the docs to become unreadable or anything